xbmc/application/Application.cpp

```diff
@@ -1041,9 +1039,7 @@ bool CApplication::OnAction(const CAction &action)
   if (action.GetID() == ACTION_CYCLE_TONEMAP_METHOD)
   {
     // Only enables tone mapping switch if display is not HDR capable or HDR is not enabled
-    if (CServiceBroker::GetSettingsComponent()->GetSettings()->GetBool(
-            CServiceBroker::GetWinSystem()->SETTING_WINSYSTEM_IS_HDR_DISPLAY) &&
-        CServiceBroker::GetWinSystem()->IsHDRDisplay())
+    if (CServiceBroker::GetWinSystem()->IsHDRDisplaySettingEnabled())
```

Comment:
Also seems odd but was already like that.

Reply:
` // Only enables tone mapping switch if display is not HDR capable or HDR is not enabled`

if is HDR display and enabled in settings tone mapping is not used. Not makes sense cycle tone map method.